Content/GUI/Tree.cs

```diff
@@ -33,16 +38,16 @@ public override void Draw(SpriteBatch spriteBatch)
 
 			if (!populated)
 			{
-				panel = new();
-				panel.Left.Set(-300, 0.5f);
-				panel.Top.Set(-400, 0.5f);
-				panel.Width.Set(600, 0);
-				panel.Height.Set(800, 0);
+				panel = new UIPanel();
+				panel.Left.Set(LeftPadding, 0);

```

Comment:
This may scale poorly with larger displays. Previously this would attempt to center the UI but this will now always put it at a specific offset from the top-left.

Reply:
With UI scale at 100% and a 4k monitor without these changes the UI actually wouldn't appear. I think this needs more investigation but this was the best i came up with. I did test other resolutions. But nothing below 1080

Reply:
That seems strange, does the issue persist after resizing the window? I know previously some UI I had worked with wouldn't properly re-size to the screen size due to forgetting to call Recalculate, possibly that could have been the issue? Would have to double check